index.js

```diff
@@ -2,7 +2,8 @@ const bencode = require('bencode')
 const ssbKeys = require('ssb-keys')
 const bfe = require('ssb-bfe')
 const ref = require('ssb-ref')
-const isCanonicalBase64 = require('is-canonical-base64')
+const SSBURI = require('ssb-uri2')
+const isCanonicalBase64 = require('iscanonicalbase64')
```

Comment:
I'm confused why was this even working before when it was imported as `is-canonical-base64`. 

[npm says this package name is `iscanonicalbase64`](https://www.npmjs.com/package/iscanonicalbase64) but [package.json name says it is `is-canonical-base64`](https://github.com/dominictarr/is-canonical-base64/blob/e46076e18e852197a382acb6d5adf60a0d5366a6/package.json#L2). :exploding_head: 

Anyway, this change is at least consistent with what we currently have in package.json "dependencies".

Reply:
Oh wow yeah, I don't know why that worked.

Looking now, it seems there are actually two separate packages for the same module? One with the hyphens and one without. The one with hyphens is `1.1.1` and the other one is `1.0.0`. I think I initially used the one with hyphens because it's used in ssb-validate (https://github.com/ssb-js/ssb-validate/blob/main/index.js#L5).

Reply:
Oh wow

Reply:
So maybe we should rather use the hyphen version in both the import statement and `package.json`?

Reply:
I'll do that

Reply:
Done